src/Microsoft.AspNet.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -50,7 +50,7 @@ public string DisplayName
 
         /// <summary>
         /// Defines whether access and refresh tokens should be stored in the
-        /// <see cref="ClaimsPrincipal"/> after a successful authentication.
+        /// <see cref="ClaimsPrincipal"/> after a successful authorization with the remote provider.
```

Comment:
nit: why replacing `authorization` by `authentication` here? The remote middleware stuff is generic enough not to be tied to OAuth2 (which is the only authorization protocol implemented by ASP.NET 5).